doc/api/stream.md

```diff
@@ -2447,6 +2450,25 @@ import { Readable } from 'node:stream';
 await Readable.from([1, 2, 3, 4]).take(2).toArray(); // [1, 2]
 ```
 
+Using the `destroyStream: false` you can use it start parsing
```

Comment:
I didn't really understand this line 😅😅😅, parsing what?

Reply:
Updated, please let me know if it's better now 😄

Reply:
Ahh understood now,
I think something like 
```
Using the `destroyStream: false` option prevents `take` from closing the stream, so that the remaining stream can be consumed later on
```
sounds better to convey what you are saying wdyt?

Reply:
sounds great